js/react/pages/admin/routes/wishcard/administration.jsx

```diff
@@ -57,6 +66,12 @@ export default function Administration() {
 		});
 	}, []);
 
+	useEffect(() => {
+		if (shouldNavigate) {
+			navigate(`/wishcards/${wishcardId}`);
+		}
```

Comment:
I think this can be moved to the forward function, there's no need to listen for a change

Reply:
Good point 👍